### PR TITLE
Add buildkite step to run on Julia LTS

### PR DESCRIPTION
**Pull request by DhairyaLGandhi**
Allow bors to unblock some PRs as well.

### CODE REVIEW

**Comment by DhairyaLGandhi** [comment]
bors try

**Review by ToucheSir — APPROVED** [review]
LGTM, do you mind sneaking in a `s/integeration/integration/g` in `pipeline.yml` while you're at it?

File: test/cuda/layers.jl [hunk 1]
```diff
@@ -50,9 +50,13 @@ function gpu_gradtest(name::String, layers::Vector, x_cpu = nothing, args...; te
               @test isnothing(xg_gpu)
             else
               if layer === GroupedConvTranspose
-                @test Array(xg_gpu) ≈ xg_cpu rtol=2f-2 atol=1f-3
+                if VERSION >= v"1.7"
+                  @test_broken Array(xg_gpu) ≈ xg_cpu rtol = 2f-2 atol = 1f-3
```
Comment (ToucheSir) [inline test/cuda/layers.jl:54]: These tests actually all pass, hence the errors on latest run. The failing one is a couple lines above at https://github.com/FluxML/Flux.jl/blob/master/test/cuda/layers.jl#L48. I think adding a conditional to mark that line as broken should be enough:
```julia
@test y_gpu ≈ y_cpu rtol=1f-3 atol=1f-3 broken=(layer === GroupedConvTranspose && args[end] === selu && VERSION >= v"1.7")
```

Comment (DhairyaLGandhi) [reply test/cuda/layers.jl:54]: I think the `broken` kwarg was recently introduced in 1.7, so I don't think we can use that generically unfortunately, and good eye on the breaking tests!

**Comment by DhairyaLGandhi** [comment]
bors try

**Comment by DhairyaLGandhi** [comment]
@ToucheSir approve?

**Review by ToucheSir — APPROVED** [review]

**Pull request merged**